WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,5 +1,8 @@
-- Added Site Menu and Home tabs to My Site screen
-- Made Add Block button more visible when block editor opens
-- Removed media/audio block upload error text
-- Added Site Intent screen to site creation process
-- Fixed crash caused by notification details for comments on GB File blocks
+* [*] App Settings: updated look and functionality of Locale Picker [https://github.com/wordpress-mobile/WordPress-Android/pull/16331]
+* [*] Fix issue that prevented Jetpack customers with standalone VideoPress subs uploading >5min videos [https://github.com/wordpress-mobile/WordPress-Android/pull/16322]
+* [*] Editor: Fix issue that caused previews on atomic sites to sometimes be out-of-date. [https://github.com/wordpress-mobile/WordPress-Android/pull/16332]
+* [*] Fixes Media Picker video selection Accessibility issue [https://github.com/wordpress-mobile/WordPress-Android/pull/16426]
+* [*] Block Editor: Latest Posts block: Add featured image settings [https://github.com/WordPress/gutenberg/pull/39257]
+* [*] Block Editor: Prevent incorrect notices displaying when switching between HTML-Visual mode quickly [https://github.com/WordPress/gutenberg/pull/40415]
+* [*] Block Editor: Embed block: Fix inline preview cut-off when editing URL [https://github.com/WordPress/gutenberg/pull/35326]
+* [*] Fixes signup magic link URL [https://github.com/wordpress-mobile/WordPress-Android/pull/16449]
```

Comment:
This one is specific to Jetpack — I kept it (but deleted the "Jetpack App" prefix [from the original line](https://github.com/wordpress-mobile/WordPress-Android/blob/0f0f347d306a30de8191dad720bb12e802bab8e6/RELEASE-NOTES.txt#L19)) in Jetpack Release Notes, but removed it from WordPress Release Notes. \cc @leandroalonso